.github/workflows/continuous-integration.yml

```
@@ -30,7 +30,7 @@ jobs:
           - php-version: "8.0"
             dbal-version: "2.13"
           - php-version: "8.1"
-            dbal-version: "3.2@dev"
+            dbal-version: "3.2"
```

If we do this, this job would become redundant: We already do test with stable DBAL 3 on the other runs. Currently, this job pulls the 3.4.x branch.

I think we should keep a job that tests with a dev version of DBAL. The ORM is heavily coupled to DBAL, so if a commit to the DBAL dev branch breaks the ORM, I wanna know. :-)

If you want to emphasize that this job does not run with DBAL 3.2 anymore, you can bump here:

```suggestion
            dbal-version: "3.4@dev"
```

… or we could remove the minor:

```suggestion
            dbal-version: "3@dev"
```

---

Oh my god, yes :facepalm: 
Let's go with the second suggestion